README.md

```diff
@@ -1,15 +1,15 @@
-# h2
-
-Currently works for youtube and pdf files, Copy any youtube or PDF url and press Ctrl/Command + Shift + V
-
+# H2
 Inspired from Helium app. 
-
+Currently works for youtube and pdf files, Copy any youtube or PDF url and press Ctrl/Command + Shift + V
```

Comment:
Can you add Google Docs as well here?

Reply:
Which google docs are you talking about? I am not getting it!

Reply:
okay sure! ^^